ansible/roles/operator-pipeline/templates/openshift/tasks/submission-validation.yml

```diff
@@ -37,7 +37,7 @@ spec:
           --pr-url $(params.git_pr_url) \
           --git-username $(params.git_username) \
           --verbose \
-          --contacts $(params.contacts)
+          --contacts $(params.github_usernames)
```

Comment:
If desired, I can update this kwarg name as well. Its description is accurate enough so I didn't bother to change it.